### PR TITLE
Steering Committee Candidate: Daniel Smith

### LINKED ISSUE #6839: Steering Committee Nomination: Daniel Smith (@lavalamp)
I'd like to nominate myself for the Steering Committee.

I have been working on Kubernetes since before it was open source. I was the top contributor for the first few years of its existence. I specialized on things I felt would enable an ecosystem to form around Kubernetes (I think that has paid off). I initiated and continue to co-run the API Machinery SIG. I would be honored to serve the community in this capacity as well.

/committee steering

### PR DESCRIPTION
**Pull request by lavalamp**
Fixes #6839

### CODE REVIEW

File: elections/steering/2022/candidate-lavalamp.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
+-------------------------------------------------------------
+```
+name: Daniel Smith
+ID: @lavalamp
+info:
+  - employer: Google
+  - slack: @lavalamp
```
Comment (coderanger) [inline elections/steering/2022/candidate-lavalamp.md:8]:
```suggestion
ID: lavalamp
info:
  - employer: Google
  - slack: lavalamp
```

Elekto will add the `@` where needed.

Comment (lavalamp) [reply elections/steering/2022/candidate-lavalamp.md]: fixed, thanks!

**Comment by lavalamp** [comment]
BTW I am OOO tomorrow (which seems to be some sort of deadline); if there are problems with this I will probably not be able to address them until early next week.

**Comment by coderanger** [comment]
> BTW I am OOO tomorrow (which seems to be some sort of deadline); if there are problems with this I will probably not be able to address them until early next week.

No worries, if there's any further formatting weirdness we can just fix it for you :)

**Comment by lavalamp** [comment]
Thanks!!

**Pull request merged**